packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1126,19 +1126,41 @@ class ScriptPopupMenuOption {
 }
 
 final defaultScriptPopupMenuOptions = [
-  copyScriptNameOption,
+  copyPackageOption,
+  copyFilePathOption,
   goToLineOption,
   openFileOption,
 ];
 
-final copyScriptNameOption = ScriptPopupMenuOption(
-  label: 'Copy filename',
+final copyPackageOption = ScriptPopupMenuOption(
```

Comment:
nit: rename to copyPackagePathOption to be consistent with the title as well as copyFilePathOption below